app-modules/task/tests/EditTaskTest.php

```diff
@@ -80,12 +92,12 @@
     // TODO: Finish these tests to ensure changes are allowed
     $request = collect(EditTaskRequestFactory::new()->create());
 
-    livewire(TaskResource\Pages\EditTask::class, [
+    livewire(EditTask::class, [
         'record' => $task->getRouteKey(),
     ])
         ->fillForm($request->toArray())
         ->call('save')
         ->assertHasNoFormErrors();
 
     // TODO: Check for changes
-});
+})->only();
```

Comment:
This `only` needs to be removed. Please be careful to ensure these are not present before requesting review.

I think all team members should do a code review of their own PRs while in Draft before opening them up for peer review. To check for things like this and to compare against everything listed in the ticket.

Reply:
Hi @Orrison,
I'm sorry about that. I'll take care of this next time.

Thank You!

Reply:
No problem!